docs/configuration.rst

```diff
@@ -39,10 +39,11 @@ Find below a list of all warnings, which can be suppressed:
 
 .. need-warnings::
 
-.. _inc_build:
+.. _`inc_build`:
```

Comment:
can we do this formatting in a different PR please, so that it does not obscure what has been changed only for schema validation documenting